build.sbt

```
@@ -52,30 +52,20 @@ javaOptions in Universal ++= Seq(
   s"-J-Xloggc:/var/log/${packageName.value}/gc.log"
 )
 
-scalaVersion := "2.12.12"
+scalaVersion := "2.13.0"
```

https://www.scala-lang.org/download/all.html
```suggestion
scalaVersion := "2.13.7"
```

---

thanks!